### PR TITLE
[MRG+1] MAINT matplotlib -> Matplotlib

### PR DESCRIPTION
**Pull request by NelleV**
Here is a useless pull request that renames matplotlib to Matplotlib everytime I found it. I only did the files that seemed the most likely to attract attention, as I think the hardest is going to be to propagate this change else where.

Feel free to commit to my branch with additional renaming.

### CODE REVIEW

File: doc/README.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-maptlotlib documentation
+Maptlotlib documentation
```
Comment (naoyak) [inline doc/README.txt]: Maptlotlib -> Matplotlib

Comment (NelleV) [inline doc/README.txt]: Thanks :)

File: doc/_templates/index.html [hunk 2]
```diff
@@ -74,17 +77,43 @@
...
 
-<p>For simple plotting the <tt>pyplot</tt> module provides a
+<p>For simple plotting the <tt>pyplot</tt> interface provides a
   MATLAB-like interface, particularly when combined
   with <tt>IPython</tt>.  For the power user, you have full control
   of line styles, font properties, axes properties, etc, via an object
   oriented interface or via a set of functions familiar to MATLAB
   users.</p>
 
+<div style="float: right; min-width: 450px; width: 50%; padding-left: 5%;">
```
Comment (tacaswell) [inline doc/_templates/index.html]: This should not be back.

File: doc/_templates/index.html [hunk 3]
```diff
@@ -115,9 +144,10 @@
 Foundation Code of Conduct</a> in everything we do.</p>
 
 <p>Check the <a href="{{ pathto('faq/index') }}">faq</a>,
-the <a href="{{ pathto('api/index') }}">api</a> docs and
-<a href="http://matplotlib.1069221.n5.nabble.com/matplotlib-users-f3.html">mailing list archives</a> for resources.
-Join the <a href="https://gitter.im/matplotlib/matplotlib">gitter</a> and the mailing lists: <a href="https://mail.python.org/mailman/listinfo/matplotlib-users">Users</a>,
+the <a href="{{ pathto('api/index') }}">api</a> docs,
+<a href="http://matplotlib.1069221.n5.nabble.com/matplotlib-users-f3.html">mailing
+list archives</a>, and join the Matplotlib
+mailing lists <a href="https://mail.python.org/mailman/listinfo/matplotlib-users">Users</a>,
```
Comment (tacaswell) [inline doc/_templates/index.html]: This drops the gitter link.

**Comment by tacaswell** [comment]
👍 except for the two text resurrections on the main page.

File: doc/_templates/index.html (hunk 2 above)
Comment (NelleV) [inline doc/_templates/index.html]: hu? What did I do wrong…

**Comment by NelleV** [comment]
I seem to be missing all of Eric's changes on the main page.

**Comment by tacaswell** [comment]
thanks!

backported to v2.x as 112d48ee4

**Pull request merged**